src/apps/lwaftr/binding_table.lua

```diff
@@ -33,15 +33,97 @@ local function pton_binding_table(bt)
    return pbt
 end
 
-function load_binding_table(bt_file)
-   if not bt_file then
-      error("bt_file must be specified or the BT pre-initialized")
+local function expand_row(range, row)
+   local v6, v4, br_v6 = row[1], row[2], row[4]
+   v6, v4 = ipv6:pton(v6), lwutil.rd32(ipv4:pton(v4))
+   if br_v6 then br_v6 = ipv6:pton(br_v6) end
+   return {v6, v4, range.start, range.last, br_v6}
+end
+
+local function coallesce(ranges)
```

Comment:
Typo: one “l” too much, it should be “[coalesce](http://www.wordreference.com/es/translation.asp?tranword=coalesce)”